Havit/Havit/Screens/Category/View/CategoryViewController.swift

```diff
@@ -144,6 +154,14 @@ class CategoryViewController: BaseViewController {
             })
             .disposed(by: disposeBag)
     }
+
+    private func setEmptyView() {
+        if categoryList.isEmpty {
+            emptyCategoryView.isHidden = false
+        } else {
+            emptyCategoryView.isHidden = true
+        }
```

Comment:
p4;

삼항연산자를 써서 간단하게 써도 좋을 듯 합니다!
```suggestion
      emptyCategoryView.isHidden = categoryList.isEmpty ? false : true
```

Reply:
@YoonAh-dev 굿 아이디어 이십니다 !! 감사합니다 ~  반영해야겠습니다 .!!! 반영 커밋 : 774bfeb

Reply:
고렇다면
emptyCategoryView.isHidden = !categoryList.isEmpty
로도 쓸 수 있을거 같아여
하지만 오히려 가독성 측면에선 삼항연산자가 더 좋은거 같기두? 이건 수연님 판단~!

Reply:
@sujinnaljin 오호... 이렇게도 쓸수 있군요 !!! 하나 더 배워갑니둥....🥑